ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -98,13 +97,11 @@ type CoinButtonsProps = {
   defaultSwapsToken?: SwapsEthToken;
   ///: END:ONLY_INCLUDE_IF
   classPrefix?: string;
-  iconButtonClassName?: string;
 };
 
 const CoinButtons = ({
   account,
   chainId,
-  trackingLocation,
```

Comment:
`trackingLocation` was used within the Receive modal and is no longer needed. All instances have been removed.

<img width="331" alt="Screenshot 2025-05-16 at 8 24 38 AM" src="https://github.com/user-attachments/assets/9472cc3b-7e99-4066-8892-5c47c37573ad" />